framework_tvb/tvb/core/services/project_service.py

```diff
@@ -760,7 +760,7 @@ def get_datatype_and_datatypegroup_inputs_for_operation(self, operation_gid, sel
         If any dataType is part of a dataType group then the dataType group will
         be returned instead of that dataType.
         """
-        all_datatypes = self._review_operation_inputs(operation_gid)[0]
+        all_datatypes = OperationService()._review_operation_inputs(operation_gid)[0]
```

Comment:
You should not call a `private` method from outside, unless some special reason.
Here, you could simply rename the OperationService method `_review_operation_inputs ` into something without leading `_`